### DISCUSSION TITLE
Cannot leave team

### DISCUSSION THREAD

**Discussion by byronpolley** [vercel/community#4724] [Account Issue] [answered]
### Question

I have a team whose SAML access was cancelled.
Now it wants me to sign in again (which I cannot do) in order to leave the team. 
Kinda counterintuitive, what to do?

**Comment by amyegan** [comment, marked as answer]
Hi @b-bot! The team's owner should be able to remove you from the team. 

Since we can't safely discuss account details in public, you should reach out to the Customer Success team if you need help logging in. You can open a ticket using one of the [direct channels](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)

**Comment by byronpolley** [comment]
@amyegan No reply from the email support.
